### DISCUSSION TITLE
Does it work with SSR?

### DISCUSSION THREAD

**Discussion by tombohub** [ant-design/ant-design-mobile#5229] [Q&A] [answered]
Does it work with SSR frameworks like Remix?
Thanks

EDIT1: I have found this: https://mobile.ant.design/guide/ssr

**Comment by awmleer** [comment, marked as answer]
The main problem with SSR is how to tell the SSR frameworks to properly load the CSS files (which is imported by js files of antd-mobile components). The behavior differs from the frameworks.

As you have posted, the documentation has a piece of [guide](https://mobile.ant.design/guide/ssr#remix) about how to use antd-mobile in Remix projects. You can try it. If does not work, please open an issue and we'll try to fix.

**Reply by tombohub** [reply]
Thanks, there is error when trying to work with Remix. I have submitted bug issue.

**Reply by awmleer** [reply]
Got it.
